src/server/editorServices.ts

```diff
@@ -754,6 +756,103 @@ namespace ts.server {
             return false;
         }
 
+        acquireTypingForJs(project: Project) {
+            if (!this.host.getTsd) {
+                return;
+            }
+            const tsd = this.host.getTsd();
+            if (project === undefined || tsd === undefined) {

```

Comment:
I see. Got cha. I miss a bit on tsd === undefined; I still would move `project === undefined` with the first if though
```ts
if (!project ||  !this.host.getTsd) {
    return;
}
const tsd = this.host.getTsd();
if (!tsd) {
    return;
}
```